sdk/cosmos/microsoft-azure-cosmos/src/test/java/com/azure/data/cosmos/rx/ChangeFeedProcessorTest.java

```diff
@@ -40,7 +50,7 @@
     private CosmosContainer createdFeedCollection;
     private CosmosContainer createdLeaseCollection;
     private List<CosmosItemProperties> createdDocuments;
-    private static Map<String, CosmosItemProperties> receivedDocuments;
+    private static volatile Map<String, CosmosItemProperties> receivedDocuments;
```

Comment:
why is this volatile?

Reply:
The initialization of the map was happening inside a Reactor thread; this is no longer the case, so I'll remove the "volatile".